po/cs.po

```diff
@@ -8,6 +8,7 @@ msgstr Ne
 
 # file: modules/fun/random/module.py:95:24
 # file: modules/fun/random/module.py:117:24
+# file: modules/fun/weeb/module.py:128:56
```

Comment:
Can you pull new version of the core and generate the .po files with that? We stopped including the string locations, they are creating too much noise.